discovery/azure/azure.go

```diff
@@ -71,7 +71,7 @@ const (
 )
 
 var (
-	userAgent = fmt.Sprintf("Prometheus/%s", version.Version)
+	userAgent = version.PrometheusUserAgent()
```

Comment:
I'd get rid of the intermediate `userAgent` vars now.
`version.PrometheusUserAgent()` is concise now and less confusing/more explicit.